rails-i18n.gemspec

```
@@ -21,4 +21,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rspec-rails", "~> 3.7"
   s.add_development_dependency "i18n-spec", "~> 0.6.0"
   s.add_development_dependency "i18n-tasks", "~> 0.9.37"
+
+  s.metadata["changelog_uri"] = s.homepage + "/blob/master/CHANGELOG.md"
```

A minor good practice in Ruby to avoid an unnecessary String instance:

```suggestion
  s.metadata["changelog_uri"] = "#{s.homepage}/blob/master/CHANGELOG.md"
```